src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useRef } from "react";
+import "twin.macro";
+import "styled-components/macro";
+import { useFormik } from "formik";
+import {
+  Button,
+  TextArea,
+  TextField,
+} from "@madie/madie-design-system/dist/react";
+import "../Functions.scss";
+import ExpandingSection from "../../../common/ExpandingSection";
+import { Checkbox, FormControlLabel } from "@mui/material";
+import { Box } from "@mui/system";
+import ConfirmationDialog from "../../common/ConfirmationDialog";
+
+export interface Funct {
```

Comment:
is there a name clash? if not can we name it `Function`?

Reply:
yeah it seemed like `function` was a reserved name. I couldn't think of anything better than `funct`. I can change it if anyone can think of something better